Cargo.toml

```diff
@@ -21,3 +21,6 @@ members = [
 # update dependencies often).
 [profile.dev.package."*"]
 opt-level = 3
+
+[patch.crates-io]
```

Comment:
I didn't even know this `patch` feature existed. 🤷

Reply:
Ah, I forgot about this! I think I used it once a while ago.